extensions/package-manager/tests/phpunit.integration.xml

```diff
@@ -19,10 +19,7 @@
     <testsuites>
         <testsuite name="Flarum Integration Tests">
             <directory suffix="Test.php">./integration</directory>
-             <exclude>./integration/tmp</exclude>
+            <exclude>./integration/tmp</exclude>
         </testsuite>
     </testsuites>
-    <php>
-        <env name="FLARUM_TEST_TMP_DIR_LOCAL" value="tests/integration/tmp" force="true" />
-    </php>
```

Comment:
Why move this from here to the composer.json?

Reply:
it's not actually moving, rather, it's no longer necessary to set the env variable since you added explicit exclusion of the vendor directory within the `tmp` directory a few months back.

The addition in `composer.json` just makes sure that it uses one of the two env variable values, or defaults back to the default value we use which is `tests/integration/tmp`